subprojects/core/src/integTest/groovy/org/gradle/api/tasks/TaskCreationBuildOperationIntegrationTest.groovy

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.tasks
+
+import org.gradle.api.internal.tasks.RealizeTaskBuildOperationType
+import org.gradle.api.internal.tasks.RegisterTaskBuildOperationType
+import org.gradle.api.specs.Spec
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.integtests.fixtures.BuildOperationsFixture
+import org.gradle.integtests.fixtures.build.BuildTestFixture
+import org.gradle.internal.operations.BuildOperationType
+import org.gradle.internal.operations.trace.BuildOperationRecord
+import org.gradle.test.fixtures.file.TestFile
+
+class TaskCreationBuildOperationIntegrationTest extends AbstractIntegrationSpec {
```

Comment:
It would be good to get some coverage somewhere that verifies that configuration actions run inside the realize build operation.

Reply:
👍

Reply:
Added.